setup.py

```diff
@@ -12,7 +12,7 @@
 setup(
     name='django-guardian',
     version=version,
-    python_requires='>=3.5',
+    python_requires='>=3.8',
```

Comment:
may want to consider `python_requires='>=3.8,<=3.12'`

Reply:
For similar reasons to https://github.com/django-guardian/django-guardian/pull/816#discussion_r1682714364, I would recommend that libraries like this not cap and upper bound on Python versions or dependencies (unless absolutely necessary).

`django-guardian` is a pure Python package with no binary wheels, so I think it's even less useful to pin upper Python bounds.

Reply:
Good to know - I actually didn't know that guardian was not packaged and distributed as a whl.  Thanks for the info!

Reply:
It's packaged as a wheel, it's just a pure Python wheel (rather than any native complied code), thus should work fine on newer versions of Python and shouldn't need an upper bound

- https://pypi.org/project/django-guardian/#files

Reply:
Leaving it as is.